service/src/test/resources/ehr/mapper/diagnosticreport/diagnostic-report-with-dummy-narrative.xml

```diff
@@ -0,0 +1,25 @@
+<component typeCode="COMP">
+    <CompoundStatement classCode="CLUSTER" moodCode="EVN">
+        <id root="II-for-DiagnosticReport-DiagnosticReport/96B93E28-293D-46E7-B4C2-D477EEBF7098"/>
+        <code code="16488004" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="laboratory reporting">
+            <originalText>Filed Report</originalText>
+        </code>
+        <statusCode code="COMPLETE"/>
+        <effectiveTime>
+            <center nullFlavor="NI"/>
+        </effectiveTime>
+        <availabilityTime value="20100225154100"/>
+            <!-- Mapped Specimen with id: Specimen/Default-1 -->
+            <component typeCode="COMP" contextConductionInd="true">
+    <NarrativeStatement classCode="OBS" moodCode="EVN">
+        <id root="5E496953-065B-41F2-9577-BE8F2FBD0757"/>
+        <text mediaType="text/x-h7uk-pmip">CommentType:DUMMY
```

Comment:
Are we literally meant to use the text "DUMMY" in the placeholder NarrativeStatement or is there some other content that should be populated here?

Reply:
There may be something in the mapping spreadsheet about this. Is row 31 relevant?

Reply:
See @matthewdoreilly comment for the correct output https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/236#discussion_r637828291

Reply:
I'll fix that according to Matthew's comment.